src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/LPG2RDFConfigurationImpl.java

```diff
@@ -54,13 +53,13 @@ public String unmapEdgeLabel(final Node node) {
     }
 
     @Override
-    public Node mapProperty(final String property) {
-        return propertyMapping.map(property);
+    public Node mapProperty(final String propertyName) {
```

Comment:
I changed propertyMapping to propertyNameMapping in other parts of the code, but I was unsure about three methods in this class(mapProperty, unmapProperty, and mapsToProperty). Is it necessary to change the names of these methods and their names in the superclass(LPG2RDFConfiguration) or not?

Reply:
Thanks for asking! I think that these method names can remain as they are, for the moment at least.